x-pack/solutions/observability/plugins/slo/server/services/get_slo_health.ts

```diff
@@ -101,36 +191,27 @@ export class GetSLOHealth {
     return summaryDocsById;
   }
 
-  private async getTransformStatsForSLO(item: {
-    sloId: string;
-    sloRevision: number;
-  }): Promise<Dictionary<TransformGetTransformStatsTransformStats>> {
-    const rollupTransformStats =
+  private async getTransformStats(from: number = 0): Promise<{
+    data: Dictionary<TransformGetTransformStatsTransformStats>;
+    count: number;
+    from: number;
+  }> {
+    const transformStats =
       await this.scopedClusterClient.asSecondaryAuthUser.transform.getTransformStats(
         {
-          transform_id: getSLOTransformId(item.sloId, item.sloRevision),
+          transform_id: 'slo-*',
```

Comment:
I was thinking about this `slo-*`.

This approach pages over all transforms matching [slo-*]. If there are many unrelated transforms, it's more work than necessary. Can we try to fetch only transforms for the SLOs discovered earlier?